src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -134,9 +137,53 @@ NTFS accessibility value | Netex accessibility value
 1 | `true`
 2 | `false`
 
-### monomodalStopPlace and multimodalStopPlace
+### StopPlace
+`StopPlace` is of 2 different types:
+- `LMO` (__Lieu d'arrêt mono-modal__): Mono-modal with a unique name, regrouping
+  `Quay`
+- `LMU` (__Lieu d'arrêt multi-modal__): Multi-modal, regrouping multiple
+  `StopPlace` of type `LMO` or `PM`
+
+`stop_area` can be associated to several `physical_mode`.  All the
+`physical_modes` need to be mapped to the Netex list (see [NeTEx Transport
+Modes](#netex-transport-modes)).  For each mode, a `StopPlace` of type `LMO` is
+created.  An additionnal `LMU` is created to regroup them.
+
+#### StopPlaceType mapping
+The `StopPlace/StopPlaceType` is defined from its `StopPlace/TransportMode`.
+
+NeTEx Mode | StopPlaceType
+--- | ---
+air | Airport
+bus | busStation
+cableway | liftStation
+coach | coachStation
+funicular | railStation
+metro | metroStation
+rail | railStation
+tram | tramStation
+water | ferryStop
+
+#### Monomodal StopPlace
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+StopPlace/@id | stops.txt | stop_id | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace).
+StopPlace/@version | | | fixed value `any`.
+StopPlace/Name | stops.txt | stop_name | 
+StopPlace/ParentSiteRef | | | link to the corresponding Multimodal `StopPlace`
+StopPlace/TransportMode | | | use the only NeTEx mode
+StopPlace/Centroid/Location | stops.txt | stop_lat and stop_lon | see [Coordinates conversion](#coordinates-conversion)
+StopPlace/StopPlaceType | | | see the section [StopPlaceType mapping](#stopplacetype-mapping)
+
+#### Multimodal StopPlace
 
-warnings: 
-- in monomodalStopPlace: all stop_points should have the same name. If not, 2 StopPlace must be created,
-- multimodalStopPlace: show the higher rated mode value only
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+StopPlace/@id | stops.txt | stop_id | see the section [Identifiers for Quay and StopPlace](#identifiers-for-quay-and-stopplace).
+StopPlace/@version | | | fixed value `any`.
+StopPlace/Name | stops.txt | stop_name | 
+StopPlace/TransportMode | | | use the mode of __highest priority__ (see [NTFS specifications](https://github.com/CanalTP/ntfs-specification/blob/v0.11.0/ntfs_fr.md#physical-modestxt-requis))
```

Comment:
```suggestion
StopPlace/TransportMode | | | use the mode of __highest priority__ (see [NTFS specifications](https://github.com/CanalTP/ntfs-specification/blob/v0.11.0/ntfs_fr.md#physical_modestxt-requis))
```

Reply:
Done in 4883c6ac2b2dddf1aa828365c0d3e5027ff70b45.